src/param.rs

```diff
@@ -1316,7 +1316,36 @@ impl AudioParamProcessor {
                                     diff,
                                     next_block_time,
                                 );
-                                self.intrisic_value = value;
+
+                                let diff = end_value - value;
+
+                                // abort event if diff is subnormal
+                                if diff.is_subnormal() || diff == 0. {
```

Comment:
I added this `diff == 0.` check so that it would work with CPU architecture that may not support subnormal (...but does it even exists?). Would be interesting to see the behaviour on your ARM machine

Reply:
It seems that my computer does support subnormal numbers. I wrote a small test file but I cannot guarantee that the compiler is doing something magical to avoid sending the f32 calculations to the hardware directly.
I am just so confused about subnormal numbers. They cannot be represented by f32, no? So how can we have a Vec<f32> and have code like this
```
  let diff = end_value - *v;

  if diff.is_subnormal() {
      *v = end_value;
  }
```
if end_value = 1.0 and *v is the next f32 value after 1.0, the diff will be `f32::EPSILON` which is greater than `f32::MIN_VALUE` meaning the check will be useless for almost all cases?

Reply:
> It seems that my computer does support subnormal numbers.

ok good to know, so probably it's not a good idea to rely on such check to do something else than flushing to zero

> I am just so confused about subnormal numbers

Héhé yeah me too :) but I'm sure I have seen some of them them on my Mac, in the new test the diff was actually "some non zero thing" and `subnormal()` was true, so we must be careful to flush subnormal to zero or perfs could be spoiled on some platforms. From what I understand (...quite badly I have to confess) with fast math compile flag or functions this problem would disappear but it seems to be a nightly feature for now (e.g. https://doc.rust-lang.org/std/intrinsics/fn.fmul_fast.html)

>  we define SNAP_TO_TARGET = 1E-10 or something and avoid all the extra code around subnormal stuff

Ok let's go like that then!